src/highdicom/sr/templates.py

```diff
@@ -2386,7 +2386,7 @@ def __init__(
         finding_sites: Optional[Sequence[FindingSite]] = None,
         method: Optional[Union[CodedConcept, Code]] = None,
         properties: Optional[MeasurementProperties] = None,
-        referenced_images: Optional[Sequence[SourceImageForMeasurement]] = None,
+        referenced_images: Optional[Sequence[Union[SourceImageForMeasurement, CoordinatesForMeasurement]]] = None,
```

Comment:
I created another option and used the same `referenced_images` parameter.

Another solution I see would be to expand `SourceImageForMeasurement` and implement all options from TID 320 in it.

If a `SCOORD` is given, use row 3 and 4. If `SCOORD3D`, then make it use row 6 (I have not implemented this). If no coordinate is passed, use row 1 (which is the current SourceImageForMeasurement functionality)

Reply:
> Then you would need to document and enforce that using referenced_images and referenced_coordinates are mutually exclusive.

I don't think this is the case.
See row 13 of TID 300. Measurement, it has `VM 1-n`, so I understand that `referenced_images` and `referenced_coordinates` can both be used inside a single `Measurement`.

Reply:
Hmm, yes I guess you are right. They are mutually exclusive within TID 320, but there's no requirement that multiple uses of TID320 within TID300 have the same type